meta-mbl-distro/recipes-core/mbl-initramfs-init/files/initramfs-init-script.sh

```diff
@@ -75,11 +29,14 @@ fi
 
 echo "Booting from init script in initramfs"
 
-# Workaround findfs failure on Raspberry Pi 3: unable to resolve 'LABEL=rootfs1'
-sleep 0.1
+# Workaround findfs failure on Raspberry Pi 3: unable to resolve 'LABEL=xxxx'
```

Comment:
As discussed - explain a little bit more about why this is needed - i.e. waiting until the driver is loaded